packages/wrangler/src/pages/dev.tsx

```diff
@@ -526,6 +526,7 @@ export const Handler = async ({
 			persist,
 			persistTo,
 			showInteractiveDevSession: undefined,
+			inspect: true,
```

Comment:
This does not need to be explicitly set - it defaults to true. When set to true, it gives the "passing --inspect is not necessary message" when using pages dev, which is rather annoying

Reply:
Looks like that's defaulting to `false` to me

Reply:
The options parameter which is destructured just after that overwrites it

Reply:
Ah, `inspect: undefined` works best: implicitly enables the inspector and means we don't get the message.